ci/deps/travis-36.yaml

```diff
@@ -11,10 +11,10 @@ dependencies:
   - gcsfs
   - geopandas
   - html5lib
-  - matplotlib
+  - matplotlib=3.0.0
```

Comment:
is there a reason this is pinned exactly to 3.0.0, which was superseded immedately by 3.0.1?

Reply:
There were almost three months between 3.0.0 and 3.0.1, see https://github.com/matplotlib/matplotlib/releases.

I just tried to spread some pins throughout the 3.x Series, see OP.